mobile-app/app/screens/AppNavigator/screens/Dex/CompositeSwap/CompositeSwapScreenV2.tsx

```diff
@@ -571,19 +571,30 @@ export function CompositeSwapScreenV2({ route }: Props): JSX.Element {
     );
     if (selectedTokenA !== undefined) {
       if (selectedTokenA.displaySymbol === "DUSD") {
-        message = translate(
-          "screens/CompositeSwapScreen",
-          "You are buying dtokens at 5% more than the oracle price at settlement block"
-        );
+        if (selectedTokenB === undefined) {
+          message = translate(
+            "screens/CompositeSwapScreen",
+            "You are buying dTokens at 5% more than the oracle price at settlement block."
```

Comment:
Hey, missed this one, need to update this to dynamic tokens

Reply:
but when tokenB empty? wont it make sense just let it to dTokens?

Reply:
right, thought it is tokenA